template-base-3/template/src/components/common/map/Map.js

```diff
@@ -1,6 +1,6 @@
 import { lazy } from 'react';
 import { useSelector } from 'react-redux';
-import 'mapbox-gl/dist/mapbox-gl.css';
+import 'maplibre-gl/dist/maplibre-gl.css';
```

Comment:
I guess that this line should be in DeckGLComponent, because it's also in the GoogleMapsComponent (then when gmaps comp is rendered, it will load maplibre-gl styles twice). Or maybe it should be only in Map.js.